commands/push-image.ts

```diff
@@ -1,14 +1,29 @@
 import vscode = require('vscode');
 import { ImageItem, quickPickImage } from './utils/quick-pick-image';
 import { reporter } from '../telemetry/telemetry';
+import { DockerNode } from '../explorer/dockerExplorer';
 const teleCmdId: string = 'vscode-docker.image.push';
 
-export async function pushImage() {
+export async function pushImage(context?: DockerNode) {
+    let imageToPush: Docker.ImageDesc;
+    let imageName: string = "";
 
-    const selectedItem: ImageItem = await quickPickImage();
-    if (selectedItem) {
-        const terminal = vscode.window.createTerminal(selectedItem.label);
-        terminal.sendText(`docker push ${selectedItem.label}`);
+    // if invokde from the explorer we have the name of the image
```

Comment:
Typo with "invoke", though we can probably just remove this comment entirely, since every command handler now has the same logic in it.